src/pages/TermsOfService.js

```diff
@@ -60,6 +53,15 @@ const TermsOfServicePage = () => {
     }
   }
 
+  const reject = async () => {
+    try {
+      await Ajax().User.rejectTos()
+    } catch (error) {
+      reportError('Error rejecting TOS', error)
```

Comment:
These error messages are user facing, so suggest spelling out "terms of service" instead of "TOS".